tests/config-wrapper-malloc-0-null.h

```diff
@@ -0,0 +1,39 @@
+/* config.h wrapper that forces calloc(0) to return NULL.
+ * Used for testing.
```

Comment:
Why not put this file in `configs/` or, if that's not acceptable, `tests/configs/`? Having it at the `tests/` root is a bit messy, IMO.

Reply:
This is not a maintained configuration, or a meaningful configuration for users. So it should be under `tests`. I can move it to a subdirectory if you prefer but I don't think this makes `tests` any less messy.

Reply:
Subdirectory would be good.

Scope creep: Do we have any other configs in the `configs` folder that are really just for testing and wouldn't really be meaningful configurations for users?

Reply:
The other configurations are useful for users. I'll create a `test/configs`.

Reply:
Done in force-push.